save-frontend/src/main/kotlin/org/cqfn/save/frontend/utils/Utils.kt

```diff
@@ -86,3 +88,12 @@ internal fun RBuilder.multilineTextWithIndices(text: String) {
         }
     }
 }
+
+/**
+ * @param cleanupCallback
+ */
+internal fun EffectBuilder.cleanup(cleanupCallback: Cleanup) {
```

Comment:
`w: /home/runner/work/save-cloud/save-cloud/save-frontend/src/main/kotlin/org/cqfn/save/frontend/utils/Utils.kt: (95, 28): Extension is shadowed by a member: public final inline fun cleanup(noinline block: Cleanup /* = () -> Unit */): Unit`